run.cc

```diff
@@ -128,4 +130,4 @@ int main(int argc, char *argv[]) {
   std::cout << "End of run.cc" << std::endl;
   delete coll;
   return 0;
-}
+}
```

Comment:
This suggests me you are not using `clang-format`. As a general rule, please use one of the methods described in [CONTRIBUTORS.md](https://github.com/bayesmix-dev/bayesmix/blob/master/CONTRIBUTORS.md#style) so that we can have clean code and pull requests